### PR TITLE
Have lights, will Blinken!

### PR DESCRIPTION
**Pull request by AdamLaurie**
use LEDs for status (and add firmware jamming). From the README:
  - Orange solid: firmware running
  - Orange blinking: packet sniffer running
  - Green momentary: packet RX
  - Red solid: ERROR - bootloader
  - Red momentary: packet TX
  - Red&Green solid: jammer active
  - Blue solid: ERROR - capture data underrun
  - Blue blinking: ERROR - USB timeout

### CODE REVIEW

File: tools/zbjammer [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+signal.signal(signal.SIGINT, interrupt)
+if not kb.is_valid_channel(args.channel):
+    print >>sys.stderr, "ERROR: Must specify a valid IEEE 802.15.4 channel for the selected device."
+    kb.close()
+    sys.exit(1)
+kb.set_channel(args.channel)
+
+print "zbjammer: jamming channel", args.channel
+print "*** WARNING: this may not actually work on your hardware! Check with spectrum analyser!"
+if not kb.jammer_on():
```
Comment (rmspeers) [inline tools/zbjammer]: The dev layers _should_ complain if the jamming feature isn't implemented on the connected hardware due to KBCapabilities being checked. Did you see that not happening or is the point that you should validate anyways with an analyzer (which I agree with!) to ensure it's happening.

File: tools/zbjammer [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+"""
+zbjammer: enable Zigbee jamming (firmware option)
```
Comment (rmspeers) [inline tools/zbjammer]: I'm thinking we should add a brief thing here like "You are responsible for using this in accordance with local laws and regulations." or something.

File: tools/zbjammer (hunk 1 above)
Comment (AdamLaurie) [reply tools/zbjammer]: Yes, the point is that you should validate anyways. I am testing this on a RZUSBStick but as per my notes in the code, it does not appear to be working despite datasheets etc. claiming it should. I will follow up with AVR and see if we can get some help.

File: tools/zbjammer (hunk 2 above)
Comment (AdamLaurie) [reply tools/zbjammer]: I don't disagree - feel free to add/edit as you see fit! :)

Comment (AdamLaurie) [reply tools/zbjammer]: Done!

Comment (rmspeers) [reply tools/zbjammer]: Sorry I didn't get to it first!

**Pull request merged**